notebook/templates/page.html

```diff
@@ -98,8 +98,18 @@
           }
       })
 
-    document.nbjs_translations = {{ nbjs_translations|safe }};
-    document.documentElement.lang = navigator.language.toLowerCase();
+      {
+          // Detect Safari
+          let chromeAgent = navigator.userAgent.indexOf("Chrome") > -1;
+          let safariAgent = navigator.userAgent.indexOf("Safari") > -1;
+          if (chromeAgent && safariAgent) safariAgent = false;
```

Comment:
Sorry I meant that this line needs `{ }` around `safariAgent = false; `.  I believe that is what is causing the build failures.

Reply:
Oops. Sorry for the misunderstanding. Done.

Reply:
It looks like the build is still failing, do the tests run successfully on your mac?

Reply:
Sorry, I am not a JavaScript developer. I know just enough of the language to fix such small problems, but have no idea about the failures. I do not see why my change could possibly trigger such errors...

Reply:
Understood, I will revisit when I have some more bandwidth.